Create ntfyscratch.js

xmeroriginals opened the pull request:
Send notifications from your Scratch project to your phone, tablet, or computer using ntfy.sh

xmeroriginals committed:
Create ntfyscratch.js

Send notifications from your Scratch project to your phone, tablet, or computer using ntfy.sh

xmeroriginals committed:
format - lint

xmeroriginals committed:
Scratch used

DangoCat committed:
[Automated] Format code

xmeroriginals committed:
!format

Seigh-sword commented:
lint, problem must be caused by, transilation issues, so... you can fix it by making each block's name to, 
```code 
Scratch.translate("Block name"), 
```
***

xmeroriginals committed:
Replace "text"s to Scratch.translate("text") texts

xmeroriginals commented:
bro is this bot kidding me

Seigh-sword commented:
and also, my extension is still under review so, ya

Brackets-Coder committed:
remove format line at top

Brackets-Coder commented:
!format

DangoCat committed:
[Automated] Format code

xmeroriginals committed:
Enhance "ntfyScratch" with advanced notification features

Updated notification functionality to support advanced features including attachments, click actions, and scheduled notifications.

ampelectrecuted commented on extensions/XmerOriginals/ntfyscratch.js:
```diff
@@ -0,0 +1,204 @@
+// Name: ntfyScratch
```
scratch is a trademark; i suggest you change this to something like turbontfy or

ampelectrecuted commented on extensions/XmerOriginals/ntfyscratch.js:
```diff
@@ -0,0 +1,204 @@
+        color1: "#55bba6",
+        color2: "#358876",
+        blocks: [
+          {
```
maybe use docsURI here

ampelectrecuted commented on extensions/XmerOriginals/ntfyscratch.js:
```diff
@@ -0,0 +1,204 @@
+              },
+              CLICK: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "https://xelabs.xmeroriginals.com",
```
a bit self-promotional even though iirc another extension on the gallery did this (lily's toolbox, shovelutils, and rixxyx are literally named after the users who made them)

ampelectrecuted reviewed (commented):
just some suggestions because i'm not a formal reviewer
